projects/storefrontlib/src/cms-components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -46,6 +47,7 @@ export class AddedToCartDialogComponent implements OnInit {
         }
       })
     );
+    this.modalIsOpen = false;
```

Comment:
This line is unnecessary, as we can make boolean assignment right at property definition (same as with quantity).